src/components/content/patient-record-search/helpers/search-result.tsx

```diff
@@ -0,0 +1,48 @@
+import fhir4 from "fhir/r4";
+import { FeedbackForm } from "@/components/content/patient-record-search/helpers/feedback-form";
+import { Badge } from "@/components/core/badge";
+import { FHIRModel } from "@/fhir/models/fhir-model";
+
+type SearchResultProps<T extends FHIRModel<fhir4.Resource>> = {
+  resource: T;
+  heading?: string;
+  label: string;
+  openDetails?: (record: T) => void;
+  text?: string | JSX.Element;
+  details?: { label: string; value?: string }[];
+};
+
+export function SearchResult<T extends FHIRModel<fhir4.Resource>>({
+  resource,
+  heading,
+  openDetails,
+  label,
+  text,
+  details,
+}: SearchResultProps<T>) {
+  return (
+    <div className="ctw-patient-record-search-result">
+      <div className="ctw-flex ctw-items-end ctw-justify-between">
+        <span>
+          <button
+            type="button"
+            className="ctw-patient-record-search-result-btn"
```

Comment:
We can use `mr-1.5` instead of `&nbsp;`
```suggestion
            className="ctw-patient-record-search-result-btn mr-1.5"
```